### PR TITLE
VIH-9999 - Disable Cache in Dev AFD

### PR DESCRIPTION
**Pull request by v1ferrarij**
### JIRA link (if applicable) ###

VIH-9999 - Disable Cache in Dev AFD

### Change description ###

VIH-9999 - Disable Cache in Dev AFD

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Comment by v1ferrarij** [comment]
TODO

**Comment by v1ferrarij** [comment]
TODO

Cache for VH Apps has temporarily been disabled due to authentication issues.
Dev's to investigate and come up with a more reliable solution next sprint, cache can be re-enabled once work has been complete.

File: environments/dev/dev.tfvars [hunk 1]
```diff
@@ -235,7 +235,7 @@ frontends = [
     name           = "vh-test-web"
     custom_domain  = "vh-test-web.dev.platform.hmcts.net"
     backend_domain = ["firewall-nonprodi-palo-sdsdev.uksouth.cloudapp.azure.com"]
-
+    cache_enabled  = false
```
Comment (adusumillipraveen) [inline environments/dev/dev.tfvars:238]:
```suggestion
    cache_enabled  = false # TODO VH team to remove after analysing application headers.
```

File: environments/dev/dev.tfvars [hunk 2]
```diff
@@ -250,7 +250,7 @@ frontends = [
     name           = "vh-video-web"
     custom_domain  = "vh-video-web.dev.platform.hmcts.net"
     backend_domain = ["firewall-nonprodi-palo-sdsdev.uksouth.cloudapp.azure.com"]
-
+    cache_enabled  = false
```
Comment (adusumillipraveen) [inline environments/dev/dev.tfvars:253]:
```suggestion
    cache_enabled  = false # TODO VH team to remove after analysing application headers.
```

File: environments/dev/dev.tfvars [hunk 3]
```diff
@@ -265,7 +265,7 @@ frontends = [
     name           = "vh-admin-web"
     custom_domain  = "vh-admin-web.dev.platform.hmcts.net"
     backend_domain = ["firewall-nonprodi-palo-sdsdev.uksouth.cloudapp.azure.com"]
-
+    cache_enabled  = false
```
Comment (adusumillipraveen) [inline environments/dev/dev.tfvars:268]:
```suggestion
    cache_enabled  = false # TODO VH team to remove after analysing application headers.
```

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**